core-web/libs/portlets/edit-ema/portlet/src/lib/shared/enums.ts

```diff
@@ -12,7 +12,7 @@ export enum NG_CUSTOM_EVENTS {
     OPEN_WIZARD = 'workflow-wizard',
     DIALOG_CLOSED = 'dialog-closed',
     EDIT_CONTENTLET_UPDATED = 'edit-contentlet-data-updated',
-    URL_IS_CHANGED = 'url-is-changed'
```

Comment:
why the change? Should we add a new one instead of modifying this?

Reply:
When I added this event/function, I named it that because the ticket was about that topic
But when migrating, I realized that this event refers to create and change the language (not just changing the URL in a new language), and to have more descriptive names, I will change the name. But the behavior is the same

Reply:
Alrighty, just wanted to be sure nothing broke after this change